security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_generate_conf.php

```diff
@@ -418,7 +418,7 @@
 				    "telnet_cmds" => "no", "ignore_telnet_erase_cmds" => "yes", 
 				    "ignore_data_chan" => "no", "def_max_param_len" => 100 );
 
-if (!is_array($snortcfg['ftp_server_engine']['item'])) {
+if (!array_get_path($snortcfg, 'ftp_server_engine/item')) {
 	$snortcfg['ftp_server_engine'] = array();
 	$snortcfg['ftp_server_engine']['item'] = array();
 }
```

Comment:
Same here, use array_init_path()

Reply:
Same as above -- implemented the suggestion.